.github/workflows/_meta-build.yaml

```diff
@@ -0,0 +1,117 @@
+on:
+  workflow_call:
+    inputs:
+      app-version:
+        type: string
+        required: false
+        default: "snapshot"
+        description: "the version that should be set/used as tag for the container image"
+    secrets:
+      registry-0-usr:
+        required: true
+      registry-0-psw:
+        required: true
+
+jobs:
+  build-java:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3.0.0
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3.0.0
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+          cache: 'maven'
+
+      - name: Overwrite Version in pom.xml
+        if: ${{ ! contains(inputs.app-version, 'snapshot') }}
+        run: |-
+          mvn versions:set -DnewVersion=${{ inputs.app-version }}
+
+      - name: Build with Maven
+        run: |-
+          mvn clean
+          mvn package -Dmaven.test.skip=true -P enhance -P embedded-jetty -Dlogback.configuration.file=src/main/docker/logback.xml
+          mvn clean -P clean-exclude-wars
+          mvn package -Dmaven.test.skip=true -P enhance -P embedded-jetty -P bundle-ui -Dlogback.configuration.file=src/main/docker/logback.xml
+          mvn clean -P clean-exclude-wars
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v3.0.0
+        with:
+          name: assembled-wars
+          path: target/*.jar
+
+  build-container:
+    runs-on: ubuntu-latest
+    needs:
+      - build-java
+
+    strategy:
+      matrix:
+        distribution:
+          - apiserver
+          - bundled
+      fail-fast: false
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3.0.0
+
+      - name: Download Artifacts
+        uses: actions/download-artifact@v3.0.0
+        with:
+          name: assembled-wars
+          path: target
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1.2.0
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1.6.0
+        id: buildx
+        with:
+          install: true
+
+      - name: Available Platforms
```

Comment:
I know this is from the current workflow, but I don't think there's a reason for keeping it.